core/templates/dev/head/services/FilterService.js

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to serve as the interface for fetching and uploading
+ * assets from Google Cloud Storage.
+ */
+
+oppia.factory('FilterService', [function () {
+  return {
+    getAbbreviatedText: function (text) {
+      if (text.length > 50 ) {
```

Comment:
Remove space before `)`

Reply:
It would be better if the limit were converted to a constant, since it minimizes any errors should we change it later on.

Reply:
Done